source/Calamari.Tests/Fixtures/Manifest/InstructionBuilder.cs

```diff
@@ -25,7 +25,7 @@ public InstructionBuilder WithCalamariInstruction(string commandName, object inp
             instructions.Add(
                              new Instruction
                              {
-                                 Launcher = LaunchTools.LaunchTools.Calamari,
+                                 Launcher = Calamari.LaunchTools.LaunchTools.Calamari,
```

Comment:
Why was this change required?

Reply:
There is now a `Calamari.Tests.LaunchTools` namespace, and that is causing problems for resolving `LaunchTools` (probably compounded by the fact that that type shares a name with its enclosing namespace).

Most of the existing tests in the project are integration tests that actually invoke Calamari, and they are organized into fixtures covering different use cases.

The new tests in this PR are unit tests of the NodeExecutor, and so I put them in a location that mirror's the NodeExecutor's location in the production project, which resulted in the above namespace resolution problem.

I could move them somewhere else and avoid this change, but don't know that there is a really good place to put them, so I just went with fully scoping the types here instead.

Reply:
Gotcha, just wanted to make sure it wasn't just an arbitrary formatting change.